frontend/src/components/ResizeTableWrapper/ResizableHeader.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import { Resizable, ResizeCallbackData } from 'react-resizable';
+
+// # Styles import
+import { SpanStyle } from './styles';
+
+function ResizableHeader(
+	props: React.HTMLAttributes<any> & {
```

Comment:
can you please make a named interface here

Reply:
Created. Done